tests/blockade.py

```diff
@@ -0,0 +1,75 @@
+"""
+This blocks all HTTP requests via httplib (which includes requests).
+
+this is a fixed copy of pytest-blockage, called "blockade"
```

Comment:
Yeah, there is already a half-assed version called [pytest-blockage](https://pypi.python.org/pypi/pytest-blockage) that this is based on, I think it would make sense to release this eventually